src/inject.js

```diff
@@ -35,7 +35,7 @@ function inject() {
             }
             if (todayValue && !stop) {
                 ++currentStreak
-            }else {
```

Comment:
I'm trying to follow these guidelines: https://github.com/airbnb/javascript#blocks--cuddled-elses